packages/common/lib/common/tools.js

```diff
@@ -649,6 +649,59 @@ function getJson(urlOrPath, agent, callback) {
     }
 }
 
+async function getJsonAsync(urlOrPath, agent) {
```

Comment:
for new methods some jsdoc would be really nice

Reply:
added